### PR TITLE
Feature/make registration page

### PR DESCRIPTION
**Pull request by divineDuckk**
┆Issue is synchronized with this [Clickup task](https://app.clickup.com/t/86c017yj8) by [Unito](https://www.unito.io)

### CODE REVIEW

File: src/components/DropDown/index.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  const handleClickOutside = (event: MouseEvent) => {
+    if (
+      dropdownRef.current &&
+      !dropdownRef.current.contains(event.target as Node)
+    ) {
+      setIsOpen(false);
+    }
+  };
+
+  useEffect(() => {
```
Comment (ilyaAzarenko) [inline src/components/DropDown/index.tsx]: Бахни отдельный хук

File: src/components/DropDown/index.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  }, []);
+
+  return (
+    <div className={styles.dropdown} ref={dropdownRef}>
+      <div className={styles.dropdownHeader} onClick={toggleDropdown}>
+        {value}
+        <img
+          src={dropDownSvg}
+          alt="drop down icon"
+          className={isOpen ? styles.dropdownArrowOpen : ''}
```
Comment (ilyaAzarenko) [inline src/components/DropDown/index.tsx]: подключай cn

File: src/components/DropDown/index.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        />
+      </div>
+      {isOpen && (
+        <ul className={styles.dropdownList}>
+          {options.map((option, index) => (
+            <li
+              key={index}
+              className={styles.dropdownListItem}
+              onClick={() => handleOptionClick(option)}
```
Comment (ilyaAzarenko) [inline src/components/DropDown/index.tsx]: карирование

File: src/utils/functions/getYearsArray.ts [hunk 4]
```diff
@@ -0,0 +1,8 @@
+export const getYearsArray = () => {
+  const currentYear = new Date().getFullYear();
+  const yearsArray = Array.from(
+    { length: currentYear - 1950 + 1 },
```
Comment (ilyaAzarenko) [inline src/utils/functions/getYearsArray.ts]: 1950 ? а может 1984 ? =D
magic numbers

**Review by ilyaAzarenko — CHANGES_REQUESTED** [review]

**Review by ilyaAzarenko000 — APPROVED** [review]

**Review by ilyaAzarenko — APPROVED** [review]

**Pull request merged**